etn-ccis/blui-react-workflows
# Feature/4465 move complete reg action to registration workflow

Fixes # BLUI-4465



#### Changes proposed in this Pull Request:

- Moved complete registration action to registration workflow component
- Refactored nexScreen function logic



#### Screenshots / Screen Recording (if applicable)
- 
<img width="1792" alt="Screenshot 2023-07-19 at 2 37 13 PM" src="https://github.com/etn-ccis/blui-react-workflows/assets/120575281/ab5d6a99-b120-40a6-8962-b3c6547c3403">



#### To Test:

- yarn start:example2
- Click on Debug
- Click on Registration Workflow
- Go through registration workflow till you see the Success screen



#### Any specific feedback you are looking for?

- Please convert #364 to open once this gets merged.
